google-chrome/verify-install.sh

```diff
@@ -0,0 +1 @@
+test -d $HOME/Applications/"Google Chrome.app"
```

Comment:
I don't check for this is be installed by the user because my installation script should be one of the first things installed.  Just be aware that this pattern doesn't check for a user-installed chrome